pkg/arvo/lib/hood/kiln.hoon

```diff
@@ -598,9 +625,19 @@
         (emil sync-ud:pass (diff:give diff) ~)
       ?:  (lth num.new-weft num.old-weft)
         ~>  %slog.0^leaf/"kiln: future version {<new-weft>}, enqueueing"
+        ::  retry upgrade if not blocked anymore
+        =/  base=arak  (~(got by ark) %base)
         =.  next.rak  (snoc next.rak [(dec aeon.rail.rak) new-weft])
+        =.  ark  (~(put by ark) loc rak)
         =/  =diff  [%block loc rak new-weft blockers=(sy %base ~)]
-        (emil sync-ud:pass (diff:give diff) ~)
+        =.  vats  (emil sync-ud:pass (diff:give diff) ~)
```

Comment:
I'm not sure we still want to report a block if we're going to update the kernel.  Maybe we should move this line down to the first branch of the `?.` so it only notifies if we can't upgrade the kernel yet.

Reply:
A counterargument is that the kernel update might happen in a later event, and until then, this desk is blocked, in which case we should notify about that intermediate state.